docs/api.md

```diff
@@ -3382,6 +3382,11 @@ __Examples__
 Ractive.styleSet( 'colors.fg', '#000' );
 ```
 
+```js
+// To change a component cssData property
+Ractive.components.MyComponentName.styleSet( 'colors.fg', '#000' );
```

Comment:
Might be better to show that `MyComponentName` is a custom component that came from `Ractive.extend()` than making assumptions as to what `Ractive.components.MyComponentName` is.

```js
const MyComponent = Ractive.extend({ ... })

// Change styles for all instances of MyComponent
MyComponent.styleSet('colors.fg', '#000')
```

Reply:
OK, good point, will redo.

Reply:
Actually though, that wouldn't add the component to the global registry would it?

How about:

```js
// Create global component
Ractive.components.MyComponent = Ractive.extend({ ... })
// Change styles for all instances of MyComponent
Ractive.components.MyComponent.styleSet('colors.fg', '#000')
```

As the point was to show a completely different entry point for styleSet.

Reply:
Not all workflows use the global registry 😉 For instance, I use ES modules and Rollup and the code ressembles the following - each dependency is scoped to the depending component:

```js
// MyComponent.js
import Ractive from 'ractive'

export default Ractive.extend({ ... })

// MyOtherComponent.js
import Ractive from 'ractive'
import MyComponent from './path/to/MyComponent.js'

export default Ractive.extend({
  components: { MyComponent },
  ...
})

// YetAnotherComponent.js
import Ractive from 'ractive'
import MyComponent from './path/to/MyComponent.js'

export default Ractive.extend({
  components: { MyComponent },
  ...
})
```

It's not safe to assume everyone's using the global registry. What we can say for certain is `styleSet` is a static function that exists on the `Ractive` constructor as well as any constructor that extends from it.